libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -852,11 +852,15 @@ DNNReturnType ff_dnn_execute_model_tf(const DNNModel *model, const char *input_n
         return DNN_ERROR;
     }
 
-    if (!out_frame) {
+    if (!out_frame && model->func_type == DFT_PROCESS_FRAME) {
```

Comment:
for changes line 855 - 862, I've changed the design that out_frame should always be not NULL.